lib/output.js

```diff
@@ -1286,8 +1286,16 @@ function OutputStream(options) {
 
     // self should be AST_New.  decide if we want to show parens or not.
     function need_constructor_parens(self, output) {
-        // Always print parentheses with arguments
-        return self.args.length > 0;
+         // Always print parentheses with arguments
+         if (self.args.length > 0) return true;
+
+        // Certain expressions are safe to add constructor calling parentheses
+        return output.option("beautify") && (
```

Comment:
Add parentheses? Hmmm... that is what caused #1111 because the node was a `true` value. Instead now, I have that other problem where I don't 100% know which nodes are safe to add them or if they could occur in new statements at all.